provider/cldr/src/transform/time_zones/convert.rs

```diff
@@ -97,18 +100,28 @@ impl From<TimeZoneNames> for ExemplarCitiesV1<'_> {
                             key.push('/');
                             key.push_str(&inner_key);
                             match place_or_region {
-                                LocationOrSubRegion::Location(place) => place
-                                    .exemplar_city()
-                                    .map(|city| vec![(key, city)])
-                                    .unwrap_or_default(),
+                                LocationOrSubRegion::Location(place) => {
+                                    match bcp47_tzid_data.get(&key) {
+                                        Some(bcp47) => place
+                                            .exemplar_city()
+                                            .map(|city| vec![(bcp47.clone(), city)])
+                                            .unwrap_or_default(),
+                                        None => Default::default(),
```

Comment:
Suggestion, here and below: pull the key lookup out of this match and up after the `key.push_str(&inner_key);` on line 101

Reply:
lines 117 & 118 also appends string in the `key`. If I pull the key lookup to line 101, the key lookup would return `None` for the branch `LocationOrSubRegion::SubRegion`.